src/crimson/os/seastore/cache.h

```diff
@@ -641,10 +646,32 @@ class Cache {
     }
   };
 
+  struct trans_byextent_efforts_t {
```

Comment:
is this supposed to be `efforts_by_extent_type_t` ?

Reply:
It follows the name of `trans_efforts_t` and it measures the transactional efforts by extent, so I named it `trans_byextent_efforts_t`.